pkgs/applications/networking/browsers/vivaldi/default.nix

```diff
@@ -106,6 +114,6 @@ in stdenv.mkDerivation rec {
     license     = licenses.unfree;
     sourceProvenance = with sourceTypes; [ binaryNativeCode ];
     maintainers = with maintainers; [ otwieracz badmutex ];
-    platforms   = [ "x86_64-linux" ];
-  };
-}
+    platforms   = [ "x86_64-linux" "aarch64-linux" ];
+    };
+  }
```

Comment:
```suggestion
  };
}
```

Reply:
Sorry, got a bit messy because I forgot to disable .nix autoformatting and had to recover from a full tidy 🤕